style.css

```diff
@@ -0,0 +1,1280 @@
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 101px;
+  margin-bottom: 85px;
+}
+
+.header__logo {
+  display: flex;
+  padding: 38px 0 0 58px;
+  cursor: pointer;
+}
+
+.header__logo-loco {
+  font: normal 500 33px 'Montserrat-Regular', sans-serif;
+  letter-spacing: .04em;
+  color: #fff;
+  margin-right: 262px;
+}
+
+.header__logo-loco:hover {
+  transform: scale(1.1);
+  transition: .3s ease;
+}
+
+.header__lang {
+  padding-top: 9px;
+  cursor: pointer;
+}
+
+.header__lang-ru {
+  font: 200 22px/24px 'Montserrat-Medium', sans-serif;
+  color: #fff;
+}
+
+.header__lang:hover {
+  transform: scale(1.1);
+  transition: .3s ease;
+}
+
+.lang__vector {
+  width: 15px;
+  height: 10px;
+}
+
+.header__menu {
+  display: flex;
+  align-self: flex-end;
+  width: 102px;
+  height: 102px;
+  background: #f2f2f2;
+  cursor: pointer;
+}
+
+.header__menu:hover {
+  background-color: #d9d7d7;
+}
+
+.header__menu-icon {
+  align-self: center;
+  height: 26px;
+}
+
+.header__menu-line {
+  display: block;
+  width: 39px;
+  height: 4px;
+  background: #000;
+  margin: 6px 30px;
+}
+
+.wrap {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 39px;
+}
+
+.wrap__main-img {
+  height: 565px;
+  margin: 0 57px 80px 0;
+}
+
+.wrap__img {
+  height: 108%;
+  padding-left: 203px;
+}
+
+.wrap__content {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.wrap__heading {
+  font:700 87px/89px 'Montserrat-Bold', sans-serif;
+  letter-spacing: .4px;
+  color: #000;
+  padding-left: 3px;
+  margin-bottom: 16px;
+}
+
+.wrap__article {
+  width: 500px;
+  font: 600 16px/23px 'Montserrat-Regular', sans-serif;
+  letter-spacing: .2px;
+  color: #000;
+  opacity: .5;
+  padding-left: 3px;
+  margin-bottom: 43px;
+}
+
+.wrap__button {
+  display: flex;
+}
+
+.wrap__button-next {
+  font: 500 14px/22px 'Montserrat-Medium', sans-serif;
+  padding: 11px 44px;
+  border: 1.5px solid #000;
+  background-color: #fff;
+  cursor: pointer;
+  margin-right: 61px;
+}
+
+.wrap__button-next:hover {
+  box-shadow: 0 0 0 4px #000;
+  text-decoration: underline;
+  transition: .3s ease;
+}
+
+.wrap__button-more {
+  align-self: center;
+  font: 500 14px/23px 'Montserrat-Medium', sans-serif;
+  cursor: pointer;
+  text-decoration: none;
+  color: #000;
+}
+
+.wrap__button-more:hover {
+  border-bottom: 2px solid #000;
+  transition: .3s ease;
+}
+
+.button-more__vector {
+  padding-left: 18px;
+  width: 48px;
+}
+
+.footer__social-networks {
+  display: flex;
+  padding-left: 58px;
+  width: 81%;
+}
+
+.footer__social-button {
+  width: 60px;
+  padding-top: 21px;
+}
+
+.footer__social-button:hover {
+  transform: scale(1.1);
+  transition: .3s ease;
+}
+
+.footer__scroll {
+  width: 32px;
+  height: 50px;
+  margin: auto;
+}
+
+.footer__scroll-link-img {
+  width: 101%;
+  height: auto;
+}
+
+.footer__scroll-link-img:hover {
+  transform: scale(1.1);
+  transition: .3s ease;
+}
+
+ /*Адаптив*/
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633354)